core/services/workflows/syncer/workflow_registry.go

```diff
@@ -60,6 +55,30 @@ type GetWorkflowMetadata struct {
 	SecretsURL   string
 }
 
+type SyncStrategy string
+
+const (
+	SyncStrategyEvent          = "event"
+	SyncStrategyReconciliation = "reconciliation"
+	defaultSyncStrategy        = SyncStrategyEvent
+)
+
+func (s *SyncStrategy) Validate() error {
+	switch *s {
+	case SyncStrategyEvent:
```

Comment:
This will drop down to the error on line 72

Do: switch s {
case SyncStrategyEvent, SyncStrategyReconciliation:
    return nil
 default:
     return err
```

instead